hikari/users.py

```diff
@@ -136,6 +137,19 @@ def app(self) -> traits.RESTAware:
     def avatar_hash(self) -> undefined.UndefinedNoneOr[str]:
         """Avatar hash for the user, if they have one, otherwise `builtins.None`."""
 
+    @property
+    @abc.abstractmethod
+    def banner_hash(self) -> undefined.UndefinedNoneOr[str]:
+        """Banner hash for the user, if they have one, otherwise `builtins.None`."""
+
+    @property
+    @abc.abstractmethod
+    def accent_color(self) -> undefined.UndefinedNoneOr[colors.Color]:
+        """The custom banner color for the user, if set.
+
+        The official client will decide the default color if not set.
+        """  # noqa: D401 - Imperative mood
```

Comment:
Should we have an alias called `accent_colour`?

Reply:
I think we should because we have `hikari.colors` and `hikari.colours`.